src/operators/rasterkernel.md

```diff
@@ -0,0 +1,99 @@
+# Raster Kernel
+
+The `RasterKernel` operator computes a rectangular kernel for a pixel and its neighborhood.
+The operator can be defined as a matrix kernel with weights or as specialized kernel types, where only the size is specified.
+
+An example usage scenario is to calculate a Sobel filter to detect contours within an image.
```

Comment:
maybe use gaussian as example instead because sobel requires additional steps.

Reply:
change the example in [9a6162d](https://github.com/geo-engine/docs/pull/34/commits/9a6162d992fe9f96e6ae78691d48e333573597c7)